src/utilities/Utilities.hpp

```diff
@@ -123,9 +127,7 @@ constexpr AnyNumber getMap(
  */
 constexpr bool areAlmostEqual(float a, float b)
 {
-    // C++11 does not allow us to declare the delta as a variable within the function body
-    // NOLINTNEXTLINE(readability-magic-numbers)
-    return getAbsolute(getAbsolute(a) - getAbsolute(b)) <= 0.001F;
+    return getAbsolute(getAbsolute(a) - getAbsolute(b)) <= 0.001f;
```

Comment:
:warning: **readability-magic-numbers** :warning:
0.001f is a magic number; consider replacing it with a named constant